src/core/energy_storage/thermal_storage.jl

```diff
@@ -22,6 +22,8 @@ Cold thermal energy storage sytem; specifically, a chilled water system used to
     macrs_itc_reduction::Float64 = 0.5
     total_itc_fraction::Float64 = 0.3
     total_rebate_per_kwh::Float64 = 0.0
+    min_duration_hours::Real = 0.0
```

Comment:
@atpham88 just noting that I updated this to add these inputs to the markdown help text, so they'll appear here: https://nrel.github.io/REopt.jl/dev/

Reply:
@adfarth I removed min/max_duration_hours in thermal storage now.